atmosphere/settings/__init__.py

```diff
@@ -465,12 +465,6 @@ def create_log_path(filename):
         "schedule": timedelta(minutes=60),
         "options": {"expires": 10 * 60, "time_limit": 2 * 60}
     },
-    "generate_metrics": {
```

Comment:
For a different PR?

Reply:
Consequence of removing code, in an attempt to pass test-cases before merging.

Reply:
@steve-gregory really the issue is that the base branch (i.e. someone else) merged code that was not passing tests, so every new pr against atmo will be breaking bec of unrelated code.

I'd submit pr to fix base branch tests, and then rebase this branch on top of the fixed base branch, so your pr is focused.

Reply:
Roger that. will do that today and look for a quick review-and-:shipit: to rebase